docs/develop/01_sdk/02_Cookbook/03_Manage an account <-> KILT DID relationship/01_link.md

```diff
@@ -0,0 +1,28 @@
+---
+id: account-link
+title: Link an account to a KILT DID
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Example5 from '!!raw-loader!@site/code-examples/core_features/linking/1_account_linking.ts';
+import Example6 from '!!raw-loader!@site/code-examples/core_features/linking/2_account_linking.ts';
+
+Beyond linking a Web3 name, KILT allows DID owners to link multiple accounts under the same DID.
+These accounts are not specific to the KILT blockchain but can reference any chain within the Kusama ecosystem.
+Each account <-> DID link requires the payment of a small deposit, which is returned whenever the link is removed.
+
+For DIDs that have also claimed a Web3 name, the linking feature opens up the way to a host of possibilities, e.g., showing the Web3 name of a collator's account on the [KILT stakeboard](https://stakeboard.kilt.io/).
```

Comment:
An idea: We could move these repeated parts to separate files and import them as an element. This way, changes would only have to be applied once.

Reply:
We don't need to go deep into the cookbook, so I simply removed the intro section altogether -> https://github.com/KILTprotocol/docs/pull/99/commits/51233ee7fdc4633493d97426202db4b55fac05c7